src/components/atoms/Banner.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="bg-banner-img bg-cover p-10">
+    <div class="mx-auto container px-3">
+      <h1

```

Comment:
The h1 and p tag overlap slightly leaving a dark line
![image](https://user-images.githubusercontent.com/69208837/121696316-f427fb00-ca99-11eb-86a9-d2b315937057.png)

Reply:
yes, it is overlapping.